### ISSUE TITLE
CMS pages no longer work for screen reader users

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/MyopicVicar#2804]
Where using a screen reader and attempting to navigate beyond the first section, **screen reader users cannot access the new section content** because the SR does not 'know' that the page content has changed.

This needs an `alert`, similar to what we are (or should be) using to report search errors to the user. Without an `alert`, a SR user cannot access any of the main content beyond what loads initially, i.e. nothing beyond the first item on the "On this page" menu.

**Comment by jayto581** [comment]
@AlOneill Would it be possible to get more information for us to try and replicate the issue. 
What browser is being used?
What operating system is being used?
Which screen reader is being used?

Is this problem happening on all CMS pages or only on specific pages?

Does it occur on the help home page?
https://www.freereg.org.uk/cms/help

If it is happening on the help home page, at what point does the screen reader stop working?

What specific section on the screen does the problem occurs.
What actions and/or links were clicked? 

Please provide as much details as possible. Screen captures would be helpful.

Thanks

**Comment by DeniseColbert** [comment]
Moving to On Ice until we know the future of Refinery

**Comment by stsccfr** [comment]
Closing as Refinery/CMS is no longer used.
